### PR TITLE
WIP : feat : Added the wishlist page

### PR DESCRIPTION
**Pull request by lannisterrr**
Added 👍 Wishlisht page.
Click on wishlist icon to visit the wishlist page.

### CODE REVIEW

**Review by GuptaEesh — COMMENTED** [review]
Just add a newline at the end of wishlist .css.

**Review by abhij1607 — APPROVED** [review]
Looks good

**Review by Hamdrive — COMMENTED** [review]
Good work done, code looks clean and proper practices followed.

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                ><i class="fa fa-shopping-cart"></i
+              ></a>
+            </span>
+            <div class="badge centered-row ecom-badge">3</div>
+          </div>
+          <span class="f-7 p-h-1 pointer badge-caption">Cart</span>
+        </div>
+      </nav>
+    </header>
+    <p class="heading-3 center-text">My Cart <span class="f-8">(3)</span></p>
```
Comment (Hamdrive) [inline wishlist.html]: Why not place the text and icon inside the same parent div?
that way user can click on either and still be able to access the page, else this just makes it a label which takes up screen space

**Review by Hamdrive — COMMENTED** [review]
Good work done, code looks clean and proper practices followed.

Bug found and have higlighted the same

File: wishlist.html (hunk 1 above)
Comment (lannisterrr) [reply wishlist.html]: I am taking a mobile first approach , and in mobile there is no label . I will finalize the desktop CSS in the end.

**Pull request merged**